scripts/generate_ipfs_hashes.py

```diff
@@ -131,7 +131,12 @@ def sort_configuration_file(config: PackageConfiguration):
     # load config file to get ignore patterns, dump again immediately to impose ordering
     assert config.directory is not None
     configuration_filepath = config.directory / config.default_configuration_filename
-    yaml_dump(config.ordered_json, configuration_filepath.open("w"))
+    if config.package_type == PackageType.AGENT:
```

Comment:
since we are not using the configuration loader, we have to manually remove the `component_configurations` from `agent_config.json`.
This is because `agent_config.json` is not actually the same JSON object of the YAML file. In the future, we should have that `agent_config.json` is a JSON Object List, but that requires a refactoring of the interface.

Reply:
would it not be easier to just use configuration loader here?

Reply:
it is too invasive for the purposes of the script